test/integration/patients/worklist/worklist.js

```diff
@@ -389,6 +389,7 @@ context('worklist page', function() {
           files: getRelationship([{ id: '1' }], 'files'),
           owner: getRelationship(teamCoordinator),
           patient: getRelationship(testPatient1),
+          comments: getRelationship([{ id: uuid() }], 'comments'),
```

Comment:
even this ideally uses `getRelationship([getComment()])`

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1385/commits/d9e8c49a663caa3d5f4c03a80da4d6105d6e6d6a.